pages/review.tsx

```diff
@@ -129,17 +129,30 @@ function ApprovalList({ type, category }: { type: RequestType; category: ReviewF
       {requests.map((requestObj: any) => (
         <Box sx={{ px: 3 }} key={requestObj._id}>
           <Grid container spacing={1} sx={requestObj.approvalType === 'Manager' ? managerStyling : reviewerStyling}>
-            <Grid item xs={12} md={6} lg={7}>
+            <Grid item xs={12} md={6}>
               {type === 'Upload' && (
                 <>
-                  <Link href={`/model/${requestObj.version?.model?.uuid}`} passHref>
-                    <MuiLink
-                      variant='h5'
-                      sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.secondary.main }}
-                    >
-                      {requestObj.version?.metadata?.highLevelDetails?.name}
-                    </MuiLink>
-                  </Link>
+                  <Stack
+                    direction={{ xs: 'column', sm: 'row' }}
+                    spacing={{ xs: 0, sm: 2 }}
+                    alignItems={{ xs: 'left', sm: 'center' }}
+                    justifyContent='flex-start'
+                    sx={{
+                      [theme.breakpoints.down('sm')]: {
+                        mb: 1,
+                      },
+                    }}
+                  >
+                    <Link href={`/model/${requestObj.version?.model?.uuid}`} passHref>
+                      <MuiLink
+                        variant='h5'
+                        sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.secondary.main }}
+                      >
+                        {requestObj.version?.metadata?.highLevelDetails?.name}
+                      </MuiLink>
+                    </Link>
+                    <div>Version: {requestObj.version?.version}</div>
```

Comment:
To avoid any of the long model name issues you could move the version number into a `Chip`  alongside the approvalType chip below.
`<Chip color='primary' label=`Version: ${requestObj. version?.version}` size='small' />`